rules/do-you-optimize-tinacms-project/rule.md

```diff
@@ -0,0 +1,106 @@
+---
+type: rule
+title: Do you optimize your TinaCMS project for clarity, performance, and reliable builds?
+seoDescription: Learn effective strategies to structure and optimize your
+    TinaCMS project for clarity, performance, and successful builds.
+uri: do-you-optimize-tinacms-project
+authors:
+    - title: 'Gilles Pothieu'
+      url: https://www.ssw.com.au/people/gilles-pothieu/
+created: 2024-08-28T16:15:00.000Z
+guid: 3c59f56f-34b0-47a7-902a-718eb340d2e5
+---
+
+Structuring and optimizing your TinaCMS project is essential to achieve clarity, enhance performance, and prevent build failures. Poorly optimized projects can lead to slow site performance, increased server load, and even failed builds due to excessive or inefficient data requests.
```

Comment:
```suggestion
Setting up and optimizing your TinaCMS project properly is key to maintaining clear organization, maximizing performance, and preventing build issues. Without optimization, your site may suffer from slow load times, increased server strain, and failed builds due to inefficient data handling.
```

Not sure I love the intro sentence but can't come up with anything better than it 😅